CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # generator-dgp-web-aspnetcore
 
+## 5.0.4
```

Comment:
Ik zou er eerder 5.1.0 van maken omdat het geen bugfix versie is. Maar maakt in de praktijk niks uit voor een generator...

Reply:
aangepast!